tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -1503,6 +1504,22 @@ namespace tuplex {
         }
     }
 
+    /*!
+     * get default file extension for supported file formats
+     * @param fmt
+     * @return string
+     */
+    std::string fileFormatDefaultExtension(FileFormat fmt) {
+        switch (fmt) {
+            case FileFormat::OUTFMT_CSV:
+                return ".csv";
+            case FileFormat::OUTFMT_ORC:
+                return ".orc";
+            default:
+                throw std::runtime_error("file format not yet supported!");
```

Comment:
maybe add to debug string fmt to int conversion, so some hint is displayed. Below a similar exception is thrown.